Sample-XML/LEI Example

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<Snapshot Date="2017-12-29">
+<Instruments>
+<Equity InstrumentId="037833100" CUSIP="037833100" InstrumentCurrency="USD" Price="160.22" InstrumentName="APPLE INC A" LEI = "HWUPKR0MPOU8FGXBT394" />
```

Comment:
This needs to reflect the full suit of properties needed on an equity to pass validation and not have missing data. I recommend copying from another example file :)